pkgs/servers/http/unit/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, fetchurl
+, which
+, python
+, php71
+, php72
+, perl
+, withSSL ? true, openssl ? null
+, withIPv6 ? true
+, withDebug ? false
+}:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  version = "1.6";
+  name = "unit-${version}";
+
+  src = fetchurl {
+    url = "https://unit.nginx.org/download/${name}.tar.gz";
+    sha256 = "0lws5xpzkcmv0gc7vi8pgnymin02dq4gw0zb41jfzq0vbljxxl14";
+  };
+
+  buildInputs = [
+    which
+    python
+    php71
+    php72
```

Comment:
Does it actually need 7.1 and 7.2?

Reply:
Each version has its own module. It is possible to easily switch versions